community/cloud-foundation/templates/gcs_bucket/examples/gcs_bucket_iam_bindings.yaml

```diff
@@ -0,0 +1,31 @@
+# Example on how to use the cloud storage bucket template
+#
+# In this example, a storage bucket is created in location us-east1
+# with name test-bucket. Cloud storage bucket names should be unique globally
+# refer https://cloud.google.com/storage/docs/naming for bucket and object 
+# naming guidelines.
+#
+# IAM Policies are also applied with the `bindings` section
+# Replace the following with appropirate values
+# <FIXME:user_account_email_address>: A valid user account
```

Comment:
# <FIXME:bucket_name> needed

Reply:
added `<FIXME:bucket_name>` to all example files. Updates in commit `8969659`